src/Compiler/Microsoft.AspNetCore.Razor.Language/test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -2103,6 +2103,34 @@ public class dynamic : ComponentBase
         CompileToAssembly(generated);
     }
 
+    [IntegrationTestFact, WorkItem("https://github.com/dotnet/razor/issues/10965")]
+    public void InvalidCode_EmptyTransition()
+    {
+        // Act
+        var generated = CompileToCSharp("""
+        <MyComponent Value="Hello" />
```

Comment:
Consider using `TestComponent` to avoid the warning about non-existent component.